core/tracepoint/fixtures/classes.rb

```diff
@@ -4,5 +4,31 @@ def m
     end
     alias_method :m_alias, :m
   end
+
+  module A
+    def bar; end
+  end
+
+  class B
+    include A
+
+    def foo; end;
+  end
+
+  class C < B
+    def initialize
+    end
+
+    def foo
+      super
+    end
+
+    def bar
+      super
+    end
+  end
 end
 
+def test
```

Comment:
This is still a global. It should be scoped within `TracePointSpec`

Reply:
okay, I have updated the PR.